checker-framework-inference.iml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess this is for some IDE.
Should we list supported IDE files somewhere, to make it obvious that people can just import the project?